### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by creeperCN** [imsyy/home#109] [General]
将源码上传到虚拟空间报错
<% if (tongji) { %> <% } %>
<%- title %>

加载中

**Comment by imsyy** [comment]
https://github.com/imsyy/home/discussions/61#discussioncomment-4840642

**Reply by creeperCN** [reply]
C:\Users\lost-\Downloads\home>yarn install
yarn install v1.22.19
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
info There appears to be trouble with your network connection. Retrying...
[2/4] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.npmmirror.com/@icon-park/vue-next/-/vue-next-1.4.2.tgz: ESOCKETTIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\lost-\\Downloads\\home\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Reply by imsyy** [reply]
更换一下 `npm` 的镜像源

**Comment by creeperCN** [comment]
谢谢我已经解决问题了，还有请问DailyHotApi可以部署到虚拟主机上吗






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年3月24日(星期五) 下午2:51
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; "State ***@***.***&gt;; 
主题: Re: [imsyy/home] 求助 (Discussion #109)





 
更换一下 npm 的镜像源

**Reply by imsyy** [reply]
虚拟主机是只有 `PHP` 吧，那估计不行 😂

**Reply by creeperCN** [reply]
vercel可以吗

**Comment by creeperCN** [comment]
vercel可以吗






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年3月24日(星期五) 下午4:06
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; "State ***@***.***&gt;; 
主题: Re: [imsyy/home] 求助 (Discussion #109)





 
虚拟主机是只有 PHP 吧，那估计不行 😂

**Reply by imsyy** [reply]
应该可行，我去试试

**Reply by creeperCN** [reply]
您能出个教程吗谢谢啦

**Reply by imsyy** [reply]
如果可行的话会出的👌

**Reply by creeperCN** [reply]
谢谢您

**Reply by imsyy** [reply]
> 谢谢您

更新好了，教程在说明文档里
